pkg/proxyagent/agent/agent.go

```diff
@@ -235,3 +255,41 @@ const (
 	AgentSecretName   = "cluster-proxy-open-cluster-management.io-proxy-agent-signer-client-cert"
 	AgentCASecretName = "cluster-proxy-ca"
 )
+
+func getAgentIndendifiersAndOtherServices(clusterName string, clusterlabels map[string]string, serviceResolvers []proxyv1alpha1.ManagedProxyServiceResolver, runtimeClient client.Client) (string, []map[string]string, error) {
+	var aids []string
+	var otherServices []map[string]string
+
+	aids = append(aids, fmt.Sprintf("host=%s", clusterName))
+	aids = append(aids, fmt.Sprintf("host=%s.%s", clusterName, config.AddonInstallNamespace))
+	aids = append(aids, fmt.Sprintf("host=%s.%s.%s", clusterName, config.AddonInstallNamespace, "svc.cluster.local"))
```

Comment:
why svc.cluster.local? a k8s cluster may not use it as postfix.

Reply:
This is copy and paste from the previous code:
```
--agent-identifiers={{ list (printf "host=%s" .Values.clusterName) (printf "host=%s.%s" .Values.clusterName .Release.Namespace) (printf "host=%s.%s.%s" .Values.clusterName .Release.Namespace .Values.serviceDomain) | join "&" }}
```

The `serviceDomain` is hard code using `svc.cluster.local`. But actually, we are not using this host.

If @yue9944882 is also not using it, I guess we can remove this.

Reply:
Check the doc and saw the format is: `my-svc.my-namespace.svc.cluster-domain.example`.
May be we can keep it, in case someone want to use a hostname with '.cluster.local'?